src/harness/testcases/WINNF_FT_S_FAD_testcase.py

```diff
@@ -0,0 +1,245 @@
+          {
+            'groupId': 'exampleGroup',
+            'groupType': 'INTERFERENCE_COORDINATION'
+          }]},
+      'grants': [{
+        'id': 'SAMPLE_ID_12345',
+        'operationParam': grant_0['operationParam'],
+        'channelType': 'GAA',
+        'grantExpireTime': grant_expire,
+        'terminated': False}]}
+    dump_file_a['recordData'].append(internal_data_a)
+    device_a_id = dump_file_a['recordData'][0]['id']
+
+    dump_file_c['startTime'] = start_time
+    dump_file_c['endTime'] = end_time
+    dump_file_c['recordData'] = []
+    internal_data_c = {
+      'id': 'device_c_id',
+      'registration': {
+        'fccId': device_c['fccId'],
+        'cbsdCategory': device_c['cbsdCategory'],
+        'callSign': device_c['callSign'],
+        'userId': device_c['userId'],
+        'airInterface': device_c['airInterface'],
+        'measCapability': device_c['measCapability'],
+        'installationParam': device_c['installationParam'],
+        'groupingParam': [
+          {
+            'groupId': 'exampleGroup',
+            'groupType': 'INTERFERENCE_COORDINATION'
+          }]},
+      'grants': [{
+        'id': 'SAMPLE_ID_12345',
+        'operationParam': grant_0['operationParam'],
+        'channelType': 'GAA',
+        'grantExpireTime': grant_expire,
+        'terminated': False}]}
+    dump_file_c['recordData'].append(internal_data_c)
+    device_c_id = dump_file_a['recordData'][0]['id']
+
+    expected_path_device_a = '/activity_dump_file/%s' % device_a_id
+    expected_path_device_c = '/activity_dump_file/%s' % device_c_id
+    # Setup the Server with response body and expected path from SAS Under Test
+    response_a = self._sas_server.setupServer({'responseBody': dump_file_a, 'expectedPath': expected_path_device_a})
+    response_c = self._sas_server.setupServer({'responseBody': dump_file_c, 'expectedPath': expected_path_device_c})
+
+    activity_dump_file_a = {}
+    activity_dump_file_a['url'] = 'https://' + config['dump_config_1']['url'] + '/activity_dump_file/%s' % device_a_id
+    activity_dump_file_a['checksum'] = config['dump_config_1']['checksum']
+    activity_dump_file_a['size'] = config['dump_config_1']['size']
+    activity_dump_file_a['version'] = config['dump_config_1']['version']
+
+    activity_dump_file_c = {}
+    activity_dump_file_c['url'] = 'https://' + config['dump_config_2']['url'] + '/activity_dump_file/%s' % device_c_id
+    activity_dump_file_c['checksum'] = config['dump_config_2']['checksum']
+    activity_dump_file_c['size'] = config['dump_config_2']['size']
+    activity_dump_file_c['version'] = config['dump_config_2']['version']
+
+    full_activity_dump = {}
+    full_activity_dump['files'] = [activity_dump_file_a, activity_dump_file_c]
+    full_activity_dump['generationDateTime'] = end_time
+    full_activity_dump['description'] = 'Load SAS Test Harness with one cbsd registration and grant'
+    expected_path_dump = '/full_activity_dump_message/test'
```

Comment:
I don't think this path is in the SAS-SAS spec.

Reply:
Already changed to the correct one.